### PR TITLE
Cards Standard Variation

### LINKED ISSUE #95: Card Standard Variation
Original - https://www.esri.com/en-us/about/about-esri/americas
New - https://main--esri--aemsites.hlx.page/en-us/about/about-esri/americas
Screenshot 2024-08-12 at 3 00 40 PM

### PR DESCRIPTION
**Pull request by anu13297**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #95 

Test URLs:
- Original: https://www.esri.com/en-us/about/about-esri/overview
- Before: https://main--esri--aemsites.hlx.live/en-us/about/about-esri/americas
- After: https://cards-standard--esri--aemsites.hlx.page/en-us/about/about-esri/americas

### CODE REVIEW

File: blocks/cards/cards.js [hunk 1]
```diff
@@ -3,12 +3,10 @@ import { domEl } from '../../scripts/dom-helpers.js';
 
 export default function decorate(block) {
   block.classList.add('calcite-mode-dark');
+  block.classList.add('cardsperrow');
```
Comment (alexcarol) [inline blocks/cards/cards.js:6]: Why is this needed? If all the cards have this class is it even required?

File: blocks/cards/cards.js [hunk 2]
```diff
@@ -3,12 +3,10 @@ import { domEl } from '../../scripts/dom-helpers.js';
...
 
   const processSimpleCard = (div) => {
-    if (!block.classList.contains('simple')) {
-      return;
-    }
-    block.classList.add('cardsperrow');
+    if (!block.classList.contains('simple')) return;
```
Comment (alexcarol) [inline blocks/cards/cards.js:9]: Instead I'd suggest only caling this function when the card is "simple" instead (and same for standardCard).

Comment (anu13297) [reply blocks/cards/cards.js]: Updated

**Review by alexcarol — APPROVED** [review]
